jsonrpc/dispatcher.go

```diff
@@ -305,39 +288,32 @@ func (d *Dispatcher) handleReq(req Request) ([]byte, error) {
 	}
 
 	if err := json.Unmarshal(req.Params, &inputs); err != nil {
-		return nil, invalidJSONRequest
+
+		return nil, NewInvalidParamsError("Invalid Params")
 	}
 
 	output := fd.fv.Call(inArgs)
-	err = getError(output[1])
-	if err != nil {
-		return nil, d.internalError(req.Method, err)
+	if err := getError(output[1]); err != nil {
+		d.internalError(req.Method, err)
+		return nil, NewInternalError("Internal error")
 	}
 
 	var data []byte
+	var err error
 	res := output[0].Interface()
 	if res != nil {
 		data, err = json.Marshal(res)
 		if err != nil {
-			return nil, d.internalError(req.Method, err)
+			d.internalError(req.Method, err)
+			return nil, NewInternalError("Internal error")
 		}
 	}
+	return data, nil
 
-	resp := Response{
-		ID:      req.ID,
-		JSONRPC: "2.0",
-		Result:  data,
-	}
-	respBytes, err := json.Marshal(resp)
-	if err != nil {
-		return nil, d.internalError(req.Method, err)
-	}
-	return respBytes, nil
 }
 
-func (d *Dispatcher) internalError(method string, err error) error {
+func (d *Dispatcher) internalError(method string, err error) {
```

Comment:
It's better to name this as `log...` because it just output a log here?

Reply:
Fixed it, [Commit-link](https://github.com/0xPolygon/polygon-sdk/pull/138/commits/8dcef833eb2f8067388f23ba87a42c40f9022a40)

Reply:
Thank you!